luma/core/device.py

```diff
@@ -192,11 +194,13 @@ class linux_framebuffer(device):
         is given, the device is determined from the `FRAMEBUFFER` environmental
         variable instead. See https://www.kernel.org/doc/html/latest/fb/framebuffer.html
         for more details.
+    :param framebuffer: Framebuffering strategy, currently instances of
+        ``diff_to_previous`` or ``full_frame`` are only supported.
```

Comment:
mention it uses diff_to_previous by default.

Reply:
fixed